engine/h2shared/d_part68k.s

```diff
@@ -0,0 +1,377 @@
+**
+** Quake for AMIGA
+** d_part.c assembler implementations by Frank Wille <frank@phoenix.owl.de>
+** Transparent version for Hexen II by Szilard Biro
+**
+
+;		INCLUDE	"quakedef68k.i"
+PART_COLOR	equ	12
+
+		XREF    _r_origin
+		XREF    _r_pright
+		XREF    _r_pup
+		XREF    _r_ppn
+		XREF    _xcenter
+		XREF    _ycenter
+		XREF    _d_vrectbottom_particle
+		XREF    _d_vrectright_particle
+		XREF    _d_vrecty
+		XREF    _d_vrectx
+		XREF    _d_pzbuffer
+		XREF    _d_zwidth
+		XREF    _d_viewbuffer
+		XREF    _d_scantable
+		XREF    _d_y_aspect_shift
+		XREF    _d_pix_min
+		XREF    _d_pix_max
+		XREF    _d_pix_shift
+		XREF    _screenwidth
+		XREF    _transTable;
+
+		XDEF    _D_DrawParticle
+
+PARTICLE_Z_CLIP         equ.s   8.0             ;must match the values in d_iface.h!
+
```

Comment:
Can you remove the one extra newline?

Also can you rebase? (Haven't noticed this one and pushed two commits to master before you.)

Reply:
You mean the newline before XDEF _D_DrawParticle? I didn't consider it extra, as it separates XREFs from XDEFs, so it makes easier to see what's what.

Reply:
The the one after PARTICLE_Z_CLIP constant.